sql/core/src/main/scala/org/apache/spark/sql/execution/ExplainUtils.scala

```diff
@@ -148,7 +148,7 @@ object ExplainUtils extends AdaptiveSparkPlanHelper {
         setOpId(p)
       case other: QueryPlan[_] =>
         setOpId(other)
```

Comment:
should this `setOpId(other)` happen after we update `currentOperationID`?

Reply:
Done